randovania/layout/preset_migration.py

```diff
@@ -987,6 +987,18 @@ def _migrate_v65(preset: dict) -> dict:
     return preset
 
 
+def _migrate_v66(preset: dict) -> dict:
+    if preset["game"] == "cave_story":
+        # Exclude the items in hell from having progression.
+        # This could be very bad in multiworld
+        preset["configuration"]["available_locations"]["excluded_indices"] = [
```

Comment:
You should add these to the existing list if they're not present, instead of overwriting the existing exclusion list